src/Compiler/Microsoft.AspNetCore.Razor.Language/test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -323,6 +323,27 @@ @using Test
         CompileToAssembly(useGenerated);
     }
 
+    [Fact]
+    public void ComponentWithTupleParameter()
+    {
+        // Arrange
+
+        // Act
+        var generated = CompileToCSharp(@"
+@code {
+    [Parameter] public (int Horizontal, int Vertical) Gutter { get; set; }
+}
+
+<TestComponent Gutter=""(32, 16)"">
+</TestComponent>
+");
```

Comment:
Consider using a raw string literal:

```suggestion
        var generated = CompileToCSharp("""
@code {
    [Parameter] public (int Horizontal, int Vertical) Gutter { get; set; }
}

<TestComponent Gutter="(32, 16)">
</TestComponent>
""");
```

Reply:
The razor-compiler repo had to revert using .NET 7 because it broke some component flow stuff, so when we get that fixed up I'll happily switch to using them :).

Reply:
Ah... makes sense!